Library/ViewModels/ProjectPamphletContentViewModel.swift

```diff
@@ -58,8 +61,27 @@ public final class ProjectPamphletContentViewModel: ProjectPamphletContentViewMo
     )
     .take(first: 1)
 
-    self.loadProjectAndRefTagIntoDataSource = Signal.combineLatest(
+    let projectCreatorDetails = project
+      .take(first: 1)
+      .map(\.slug)
+      .switchMap { slug in
+        AppEnvironment.current.apiService
+          .fetchProjectCreatorDetails(query: projectCreatorDetailsQuery(withSlug: slug))
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+          .map { result in (result, false) }
+          .prefix(value: (nil, true))
+          .demoteErrors(replaceErrorWith: (nil, false))
```

Comment:
If this errors out we just return `nil` and tell the cell to stop loading. We will do additional work to not even activate the experiment if we don't have this data.